addons/account/models/account_move.py

```diff
@@ -1134,7 +1134,7 @@ def _compute_tax_totals(self):
 
                 kwargs = {
                     'base_lines': base_line_values_list,
-                    'currency': move.currency_id or move.journal_id.currency_id or move.journal_id.company_id.currency_id,
+                    'currency': move.currency_id or move.journal_id.currency_id or move.journal_id.company_id.currency_id or move.company_id.currency_id,
```

Comment:
```suggestion
                    'currency': move.currency_id or move.journal_id.currency_id or move.company_id.currency_id,
```
since company_id is computed based on journal_id

Reply:
Changes done.